img/calendar-white.svg

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I've created a PR to add white calendar icon to core in nextcloud/server#5447 - so this should become redundant